index.html

```diff
@@ -2,78 +2,144 @@
-        margin: 40px auto;
-        max-width: 800px;
-        padding: 38px 31px;
-      }
-
-      input {
-        font-size: 21px;
-        line-height: 33px;
-        margin: 0 0 23px 0;
-        padding: 0 9px;
-        width: 100%;
-      }
-
-      button {
-        font-size: 21px;
-        line-height: 33px;
-        margin: 0 0 23px 0;
-        padding: 0 6px;
-      }
-
-      #output-div {
-        background-color: lightgrey;
-        margin: 20px auto;
-        padding: 20px;
-        width: 100%;
-      }
-    </style>
+    <link rel="stylesheet" href="style.css" type="text/css" />
   </head>
 
   <body>
     <h1 id="header">Basics: Beat That! 🚀</h1>
     <div id="container">
-      <p>Input:</p>
-      <input id="input-field" />
+      <table>
+        <tr>
+          <th>Number of Players:</th>
+          <th>Number of Dice:</th>
+          <th>Dice Mode:</th>
+          <th>Game Mode:</th>
+        </tr>
+        <tr>
+          <td>
+            <input id="input-player" min="2" max="8" type="number" value="2" />
+          </td>
+          <td>
+            <input id="input-dice" min="2" max="8" type="number" value="2" />
+          </td>
+          <td>
+            <input
+              type="radio"
+              id="highest-mode"
+              name="diceMode"
+              value="highest"
+              checked
+            />
+            <label for="highest-mode">Highest</label>
+            <input
+              type="radio"
+              id="lowest-mode"
+              name="diceMode"
+              value="lowest"
+            />
+            <label for="lowest-mode">Lowest</label>
+          </td>
+          <td>
+            <input
+              type="radio"
+              id="standard-mode"
+              name="gameMode"
+              value="standard"
+              checked
+            />
+            <label for="standard-mode">Standard</label>
+            <input
+              type="radio"
+              id="knockout-mode"
+              name="gameMode"
+              value="knockout"
+            />
+            <label for="knockout-mode">Knockout</label>
+          </td>
+        </tr>
+        <tr></tr>
+        <tr>
+          <td>
+            <button id="start-button">Start!</button>
+          </td>
+          <td>
+            <button id="roll-button" disabled>Roll dice!</button>
+          </td>
+          <td>
+            <button id="reset-game" disabled>Reset game</button>
+          </td>
+        </tr>
+      </table>
+
       <br />
-      <button id="submit-button">Submit</button>
+
       <p>Output:</p>
-      <div id="output-div"></div>
+      <div id="output-div">
+        Hello and welcome to Beat That! Take turns rolling the dice and highest
+        or lowest number wins! Good luck and have fun!
+      </div>
     </div>
     <!-- Import program logic -->
     <script src="script.js"></script>
     <!-- Define button click functionality -->
     <script>
```

Comment:
this whole script from line 85 to 144 is good, and you have very solid understanding of DOM manipulation which will make transition to bootcamp very smooth for you so im happy to see that!

i think this is ok for now but if this gets too long (maybe for blackjack?), can consider putting it in a separate js file and import it like in line 83, maybe smth like ui-helper.js